substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfInfoSectionImpl.java

```diff
@@ -326,10 +1221,93 @@ public int writeAttrString(String value, byte[] buffer, int p) {
         }
     }
 
+    public int writeAttrAccessibility(int modifiers, byte[] buffer, int p) {
+        byte access;
+        if (Modifier.isPublic(modifiers)) {
+            access = DW_ACCESS_public;
+        } else if (Modifier.isProtected(modifiers)) {
+            access = DW_ACCESS_protected;
+        } else if (Modifier.isPrivate(modifiers)) {
+            access = DW_ACCESS_private;
+        } else {
+            // package private -- make it public for now
+            access = DW_ACCESS_public;
+        }
+        return writeAttrData1(access, buffer, p);
+    }
+
+    public int writeOopRelocationExpression(byte[] buffer, int p) {
```

Comment:
Aha there it is!

If you also have such code why is the explicit `& ~0x7L` necessary in the example in `DebugInfo.md`